server/service/src/sync/test/integration/errors.rs

```diff
@@ -51,6 +52,7 @@ mod tests {
     }
     #[actix_rt::test]
     async fn integration_sync_parsed_error() {
+        // util::init_logger(util::LogLevel::Warn);
```

Comment:
required?

Reply:
remove in head pr: bb8309272975b8f867a4d8f8c4cd2d87c73d93f2